packages/manager/apps/web/client/app/hosting/email/detach/detach.html

```diff
@@ -0,0 +1,27 @@
+<div data-ovh-alert="detach"></div>
```

Comment:
`alert` before back button and title ?

Reply:
Yes, I've placed them on top of the details section, in the page. As it is often the case, in services dashboards, I've choose to place them above.